onnxruntime/core/framework/session_state_initializer.cc

```diff
@@ -10,19 +10,17 @@
 #include "core/common/logging/logging.h"
 
 #include "core/graph/graph_viewer.h"
-#include "core/graph/graph_transformer.h"
-#include "core/graph/graph_transformer_mgr.h"
+#include "core/optimizer/graph_transformer.h"
+#include "core/optimizer/graph_transformer_mgr.h"
```

Comment:
Would you fix this? Framework shouldn't depend on optimizer.

Reply:
Good catch! Will change it. Thanks!